docs/api.md

```diff
@@ -604,3 +604,123 @@ Create a LinkedGeoPolygon describing the outline(s) of a set of hexagons, conver
 Splits polygons when crossing 180th meridian.
 
 
+# Raster processing functions
+
+## Continuous raster data
+Combining summary stats from multiple rasters:
+```
+SELECT
+    (summary).h3 AS h3,
+    (h3_raster_summary_stats_agg((summary).stats)).*
+FROM (
+    SELECT h3_raster_summary(rast, 8) AS summary
+    FROM rasters
+) t
+GROUP BY 1;
+```
+
+
+*Since vunreleased*
+
+
+### h3_raster_summary_stats_agg(setof `h3_raster_summary_stats`)
+*Since vunreleased*
+
+
+### h3_raster_summary_clip(rast `raster`, resolution `integer`, [nband `integer` = 1]) ⇒ TABLE (h3 `h3index`, stats `h3_raster_summary_stats`)
+*Since vunreleased*
+
+
+Returns `h3_raster_summary_stats` for each H3 cell in raster for a given band. Clips the raster by H3 cell geometries and processes each part separately.
+
+
+### h3_raster_summary_centroids(rast `raster`, resolution `integer`, [nband `integer` = 1]) ⇒ TABLE (h3 `h3index`, stats `h3_raster_summary_stats`)
+*Since vunreleased*
+
+
+Returns `h3_raster_summary_stats` for each H3 cell in raster for a given band. Finds corresponding H3 cell for each pixel, then groups values by H3 index.
+
+
+### h3_raster_summary_subpixel(rast `raster`, resolution `integer`, [nband `integer` = 1]) ⇒ TABLE (h3 `h3index`, stats `h3_raster_summary_stats`)
+*Since vunreleased*
+
+
+Returns `h3_raster_summary_stats` for each H3 cell in raster for a given band. Assumes H3 cell is smaller than a pixel. Finds corresponding pixel for each H3 cell in raster.
+
+
+### h3_raster_summary(rast `raster`, resolution `integer`, [nband `integer` = 1]) ⇒ TABLE (h3 `h3index`, stats `h3_raster_summary_stats`)
+*Since vunreleased*
+
+
+Returns `h3_raster_summary_stats` for each H3 cell in raster for a given band. Attempts to select an appropriate method based on number of pixels per H3 cell.
+
+
+## Discrete raster data
+Combining summary from multiple rasters into a single JSON object for each H3 index,
+adding `fraction` value (fraction of H3 cell area for each value):
+```
+WITH
```

Comment:
Added sample output and a description.

Reply:
Interested and innocent bystander: It would be good to have an example of this but where the return value is simpler, e.g. just the value at the H3 cell centre, or the return value is the mode value, rather than these complex objects.

Reply:
@alpha-beta-soup
Thank you, that's a good idea. I've added an example of getting the most frequent value in each cell.
The other example shows an intended use case, and I also tried to show how to calculate a total # of pixels in each cell, since it's not immediately obvious.